src/engine/Input/Pointer.ts

```diff
@@ -80,6 +86,44 @@ export class PointerEvent extends GameEvent<any> {
 };
 
 /**
+ * Wheel Events
+ *
+ * Represents a mouse wheel event. See [[Pointers]] for more information on
+ * handling point input.
+ */
+export class WheelEvent extends GameEvent<any> {
```

Comment:
We will have to take into account devices which can scroll in multiple directions at once, do we examine the magnitude and choose the highest for the "direction" enum?

Then, what do we do if deltaX and deltaY are equal, do we always favor Y movement over X movement?